docs/product/components/cards.html

```diff
@@ -1,73 +1,142 @@
+                    <p class="fs-body1 fc-medium">When vertical space is an issue, the CTA can be moved inline with paragraph text. <span a href="#url" class="s-link">Learn more</span></p>
+                </a>
+                <a href="#url" class="grid--cell s-card">
+                    <h2 class="fs-body3 lh-sm fc-dark">Linked card title</h2>
+                    <p class="fs-body1 fc-medium">When a card's content is visually complex (ex. a graph) or having extra decoration is visually distracting.</p>
+                </a>
+                <a href="#url" class="grid--cell s-card bs-hover bs-sm">
+                    <h2 class="fs-body3 lh-sm fc-dark">Small box shadow</h2>
+                    <p class="fs-body1 fc-medium">This is a description of the card's content.</p>
+                    <p class="fs-body1 s-link">Learn more</p>
+                </a>
+                <a href="#url" class="grid--cell s-card bs-hover bs-md">
+                    <h2 class="fs-body3 lh-sm fc-dark">Medium box shadow</h2>
+                    <p class="fs-body1 fc-medium">This is a description of the card's content.</p>
+                    <p class="fs-body1 s-link">Learn more</p>
+                </a>
+                <a href="#url" class="grid--cell s-card bs-hover bs-lg">
+                    <h2 class="fs-body3 lh-sm fc-dark">Large box shadow</h2>
+                    <p class="fs-body1 fc-medium">This is a description of the card's content.</p>
+                    <p class="fs-body1 s-link">Learn more</p>
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
+
 <section class="stacks-section">
     {% header h2 | Muted %}
-    <p class="stacks-copy">When a card is disabled or considered completed; apply the muted modifier to visually dim the content group.</p>
+    <p class="stacks-copy">When a card is disabled or considered completed, apply the muted modifier to visually dim the card.</p>
     <div class="stacks-preview">
 {% highlight html linenos %}
 <article class="s-card s-card__muted">
     <h1 class="fs-body3 fc-dark">…</h1>
-    <p class="fs-caption fc-light">…</p>
+    <p class="fs-body1 fc-light">…</p>
 </article>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="grid">
-                <article class="s-card s-card__muted wmx3">
-                    <h1 class="fs-body3 fc-dark">Sample card title</h1>
-                    <p class="fs-caption fc-light">This is some explanation copy to explain what to do.</p>
+            <div class="grid grid__allcells4 md:fd-column gs12">
+                <article class="grid--cell s-card s-card__muted">
+                    <h2 class="fs-body3 lh-sm fc-dark">Base card title</h2>
+                    <p class="fs-body1 fc-medium">This is a description of the card's content.</p>
                 </article>
+                <a href="#url" class="grid--cell s-card s-card__muted">
```

Comment:
Just give these dummy links in examples a `#` for the href.

Reply:
done